apps/explorer/src/ui/Longtext.tsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCallback, useState, useContext } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+
+import { ReactComponent as ContentArrowRight } from '../../assets/SVGIcons/12px/ArrowRight.svg';
+import { ReactComponent as ContentCopyIcon16 } from '../../assets/SVGIcons/16px/Copy.svg';
+import { ReactComponent as ContentCopyIcon24 } from '../../assets/SVGIcons/24px/Copy.svg';
+import { NetworkContext } from '../../context';
+import { navigateWithUnknown } from '../../utils/searchUtil';
+import ExternalLink from '../external-link/ExternalLink';
+
+import styles from './Longtext.module.css';
```

Comment:
Use CVA instead of CSS modules for UI components.

Reply:
These comments are on WIP stuff. I will let you know when this PR is ready for review.